Source/DafnyCore/Resolver/ModuleResolver.cs

```diff
@@ -2467,52 +2467,49 @@ void ProcessInheritedTraitMembers(TopLevelDeclWithMembers cl, List<string> suppr
           continue;
         }
 
-        var traitMember = member.OverriddenMember;
-        var trait = traitMember.EnclosingClass;
-        if (trait is not TraitDecl) {
```

Comment:
The removal of this test is a regression. It used to trigger the error `GeneralNewtypeMemberResolution.dfy(48,11)`, which is now missing.